plugins/module_utils/identity/bitwarden.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
This collection now follows[^1] the [REUSE specification](reuse.software/spec/). Please update the headers for the other files you added to use this format.

```suggestion
# Copyright (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
# SPDX-License-Identifier: GPL-3.0-or-later
```

[^1]: Technically, it's still in progress, as not every file is compliant.